docs/README.md

```diff
@@ -75,6 +108,229 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+    metadata:
+      name: default
+    spec:
+      namespace: istio-system
+      updateStrategy:
+        type: InPlace
+        inactiveRevisionDeletionGracePeriodSeconds: 30
+      version: v1.21.0
+    EOF
+    ```
+
+3. Confirm the control plane is installed and is using the desired version.
+
+    ```bash
+    $ kubectl get istio -n istio-system
+    NAME      READY   STATUS    IN USE   VERSION   AGE
+    default   True    Healthy   True     v1.21.0   2m
+    ```
+
+4. Get the `istiorevision` name
+
+    ```bash
+    $ kubectl get istiorevision -n istio-system
+    NAME              READY   STATUS    IN USE   VERSION   AGE
+    default-v1-21-0   True    Healthy   False    v1.21.0   114s
+    ```
+
+5. Create `bookinfo` namespace and label it with the revision name
+
+    ```bash
+    kubectl create namespace bookinfo
+    kubectl label namespace bookinfo istio.io/rev=default-v1-21-0
+    ```
+
+6. Deploy bookinfo application
+
+    ```bash
+    kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.22/samples/bookinfo/platform/kube/bookinfo.yaml
+    ```
+
+7. Confirm proxy sidecar injection version match the control plane version
+
+    ```bash
+    istioctl proxy-status 
+    ```
+The column `VERSION` should match the control plane version.
+
+8. Update the control plane to a new version
+
+    ```bash
+    kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.21.2"}}'
+    ```
+
+9. Verify the `Istio` and `IstioRevision` resources. There will be a new revision created with the new version.
+
+    ```bash
+    $ kubectl get istio -n istio-system
+      NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   VERSION   AGE
+      default   2           2       1        Healthy           v1.21.2   23m
+    $ kubectl get istiorevision -n istio-system
+      NAME              READY   STATUS    IN USE   VERSION   AGE
+      default-v1-21-0   True    Healthy   True     v1.21.0   27m
+      default-v1-21-2   True    Healthy   False    v1.21.2   4m45s
+    ```
+
+10. Confirm there are two control plane pods running, one for each revision
+
+    ```bash
+    $ kubectl get pods -n istio-system
+      NAME                                      READY   STATUS    RESTARTS   AGE
+      istiod-default-v1-21-0-69d6df7f9c-grm24   1/1     Running   0          28m
+      istiod-default-v1-21-2-7c4f4674c5-4g7n7   1/1     Running   0          6m9s
+    ```
+
+11. Confirm the proxy sidecar version remains the same:
+
+    ```bash
+    istioctl proxy-status 
+    ```
+The column `VERSION` should still match the old control plane version.
+
+12. Change the label of the `bookinfo` namespace to use the new revision
+
+    ```bash
+    kubectl label namespace bookinfo istio.io/rev=default-v1-21-2 --overwrite
+    ```
+The injected version by the control plane will not change immediately. Only will be changed after the pods are restarted, this will trigger the sidecar injection with the new version.
```

Comment:
```suggestion
The injected version by the control plane will not change immediately. It will only be changed after the pods are restarted, this will trigger the sidecar injection with the new version.
```